app/helpers/verification_helper.rb

```diff
@@ -293,7 +293,10 @@ def ridp_admin_actions(ridp_type, person)
   end
 
   def build_admin_actions_list(v_type, f_member)
-    if f_member.consumer_role.aasm_state == 'unverified' || VerificationType::ADMIN_CALL_HUB_VERIFICATION_TYPES.exclude?(v_type.type_name)
+    if EnrollRegistry.feature_enabled?(:ai_an_self_attestation) && v_type.type_name == VerificationType::AMERICAN_INDIAN_STATUS
+      attested_only = ["Verify", "Reject", "Call HUB", "Extend"]
+      ::VlpDocument::ADMIN_VERIFICATION_ACTIONS.reject{ |action| attested_only.include?(action) }
```

Comment:
for readability sake, I'd offer to use the `-` operator instead of `reject`, but super nitpicky